addons-cluster/etcd/values.schema.json

```diff
@@ -0,0 +1,160 @@
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": ["kubernetes.io/hostname"]
+    },
+    "affinity": {
+      "title": "Affinity",
+      "description": "Affinity setting for etcd cluster pods assignment",
+      "type": "object",
+      "default": {}
+    },
+    "replicas": {
+      "title": "Replicas",
+      "description": "Number of replicas",
+      "type": "integer",
+      "default": 3
+    },
+    "peerService": {
+      "title": "Peer Service",
+      "description": "Configuration for etcd peer communication, not required to setup peerService if you're using a single k8s cluster",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Peer Service Name",
+          "description": "When name is not empty, peerService will be enabled",
+          "type": "string",
+          "default": ""
+        },
+        "type": {
+          "title": "Peer Service Type",
+          "description": "Etcd peerService type, recommended option is LoadBalancer",
+          "type": "string",
+          "default": "LoadBalancer"
+        },
+        "annotations": {
+          "title": "Peer Service Annotations",
+          "description": "Optional annotations for Cloud Provider if peerService type is LoadBalancer",
+          "type": "object",
+          "default": {}
+        }
+      }
+    },
+    "clientService": {
+      "title": "Client Service",
+      "description": "Configuration for etcd client communication",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Client Service Name",
+          "description": "When name is not empty, clientService will be enabled",
+          "type": "string",
+          "default": ""
+        },
+        "type": {
+          "title": "Client Service Type",
+          "description": "Etcd service type, valid options are ExternalName, ClusterIP, NodePort, and LoadBalancer",
+          "type": "string",
+          "default": "ClusterIP"
+        },
+        "port": {
+          "title": "Client Service Port",
+          "description": "Port for clientService",
+          "type": "integer",
+          "default": 2379
+        },
+        "role": {
+          "title": "Client Service Role",
+          "description": "Role for clientService",
+          "type": "string",
+          "default": "leader"
+        },
+        "nodePort": {
+          "title": "Client Service NodePort",
+          "description": "Optional nodePort if clientService type is NodePort",
+          "type": ["integer", "null"],
+          "default": null
+        },
+        "annotations": {
+          "title": "Client Service Annotations",
+          "description": "Optional annotations for Cloud Provider if clientService type is LoadBalancer",
+          "type": "object",
+          "default": {}
+        }
+      }
+    },
+    "resources": {
```

Comment:
Better to refer other addon cluster chart, set its cpu/memory and storage.

Reply:
Since resources.limits[requests].cpu[mem, storage] = 0 means that no limit, so I use 0 as default.

Reply:
I think @ldming means the format of etcd/values.shcema.json should follow other addons' values.shcema.json

Reply:
so that the parameters of `kbcli cluster create etcd` can be similar to other addons.
e.g. 
kbcli cluster create etcd --cpu=1 --memory=2 --storage=30

Reply:
fixed